apps/api/src/utils/test/seed/employee.ts

```diff
@@ -0,0 +1,20 @@
+import { Employee } from '../../../models/employee/EmployeeEntity';
+import { Connection } from 'typeorm';
+import { plainToInstance } from 'class-transformer';
+import { v4 as uuidv4 } from 'uuid';
+import { EnumUserRole } from '@innbyggerpanelet/api-interfaces';
+
+export const createDummyEmployee = async (connection: Connection): Promise<Employee> => {
+    const repository = connection.getRepository(Employee);
+
+    return await repository.save(repository.create(plainToInstance(Employee, {
+        id: uuidv4(),
+        firstname: 'Dan',
+        surname: 'Børge',
+        registered: true,
+        role: EnumUserRole.InsightWorker,
+        messages: [],
+        email: 'dan.børge@nav.no',
+        insightProjects: []
+    })));
+};
```

Comment:
Might want to define the Employee outside of the `repository.save()` function in case we want to add several as an array later on.